test/config/security/peerauthentication.yaml

```diff
@@ -1,46 +0,0 @@
-# Copyright 2019 The Knative Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-apiVersion: "security.istio.io/v1beta1"
-kind: "PeerAuthentication"
```

Comment:
Is this needed for mTLS test?

Reply:
No, that's no problem to remove it. mTLS setting was included in net-istio and enabled during istio installation https://github.com/knative-sandbox/net-istio/pull/309/files#diff-a9404b8b01d51711b3d09fd67af8cc16fc687ffaf832539291c38323f5147f3dR32-R35.